### PR TITLE
GER redesign setup

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@sewagodimo please review

File: gem/templates/rwanda/core/footer_page.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<div class="articles nav">
+  <h3 class="article-title">{{self.title}}</h3>
+  <p>{{self.subtitle}}</p>
+
+  {% if self.image %}
+  <div class="banner">{% image self.image width-320 %}</div>
+  {% endif %}
+
+<article>
+  {% for block in self.body %}
```
Comment (sewagodimo) [inline gem/templates/rwanda/core/footer_page.html]: this this a django template block instance? `{%block content %} {%endblock%}`
I.E. are we getting all the blocks in the template body and rendering them?
If not the word choice is a bit confusing

File: gem/templates/rwanda/core/footer_page.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      {% if block.block_type == 'heading' %}
+          <h1>{{ block.value }}</h1>
+      {% else %}
+      {% if block.block_type == 'image' %}
+        {% image block.value width-320 %}
+      {% else %}
+      {% if block.block_type == 'numbered_list' %}
+        <ol>
+        {% for item in block.value %}
```
Comment (sewagodimo) [inline gem/templates/rwanda/core/footer_page.html]: item could just be named value

File: gem/templates/rwanda/core/footer_page.html (hunk 1 above)
Comment (Mitso) [reply gem/templates/rwanda/core/footer_page.html]: @sewagodimo is currently a clone of the base template (initial setup) - will be fixed on subsequent tickets

File: gem/templates/rwanda/core/footer_page.html (hunk 2 above)
Comment (Mitso) [reply gem/templates/rwanda/core/footer_page.html]: Same as above comment

File: gem/templates/rwanda/core/footer_page.html (hunk 1 above)
Comment (Mitso) [reply gem/templates/rwanda/core/footer_page.html]: Updated

**Pull request merged**